modules/oe_whitelabel_search/oe_whitelabel_search.post_update.php

```diff
@@ -19,3 +19,16 @@ function oe_whitelabel_search_post_update_00001(&$sandbox) {
   $block->set('settings', $settings);
   $block->save();
 }
+
+/**
+ * Set default button label in search block if it's not already set.
+ */
+function oe_whitelabel_search_post_update_00031(&$sandbox) {
+  $block = Block::load('oe_whitelabel_search_form');
```

Comment:
We need to set this setting in current theme search block instances rather than the block provided by the theme, sites will have its own configuration for the search block as oesa does with oe_showcase_theme.
To do so the logic is the same but we have to:

1. Get blocks from entity type manager, and filter theme (current) and plugin ('whitelabel_search_block').
2. Check if there is value, if its we skip, if not we set the value.